src/DynamoCore/Graph/Workspaces/WorkspaceModel.cs

```diff
@@ -29,6 +29,72 @@
 namespace Dynamo.Graph.Workspaces
 {
     /// <summary>
+    /// Non view-specific container for additional view information required to
+    /// fully construct a WorkspaceModel from JSON
+    /// </summary>
+    public class ExtraViewInfo
```

Comment:
I have updated the names from ExtraViewInfo to ExtraWorkspaceViewInfo, including some argument names in other places for consistency